src/Disclosure/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react'
+import { DisclosureProps } from './interface'
+import {
+  TitleContainer,
+  MainContainer,
+  ContentContainer,
+  StyledCaretIcon,
+} from './Disclosure.style'
+
+const Disclosure: React.FC<DisclosureProps> = ({ ...props }) => {
+  const { title, content } = props
```

Comment:
destructure dalam 1 line aja 
```suggestion
const Disclosure: React.FC<DisclosureProps> = ({  title, content }) => {
```

Reply:
done